lib/matplotlib/collections.py

```diff
@@ -1302,12 +1316,14 @@ def _add_offsets(self, segs):
 
     def set_color(self, c):
         """
-        Set the color(s) of the line collection.  *c* can be a
-        matplotlib color arg (all patches have same color), or a
-        sequence or rgba tuples; if it is a sequence the patches will
-        cycle through the sequence.
+        Set the color(s) of the line collection.
```

Comment:
Pure curiosity from myself: is the choice of “line collection” instead of "LineCollection" as in the other docstrings something intentional?